app/controllers/rating_questions_controller.rb

```diff
@@ -1,5 +1,70 @@
+# require 'pry'
+
 class RatingQuestionsController < ApplicationController
+  before_action :set_rating_question, only: [:show, :update, :destroy]
+
   def index
     @rating_questions = RatingQuestion.all
   end
+
+  def new
+    @rating_question = RatingQuestion.new
+  end
+  
+  def question_params
+    params.require(:rating_question).permit(:title, :tag)
+  end
+
+  def create
+   if request.body.size.zero?
+    return render json: {}, status: 400
+   end
+    @rating_question = RatingQuestion.create(question_params)
+    if @rating_question.save
+      respond_to do |format|
+        format.html { redirect_to "/rating_questions", notice: "Your question has been created." }
+        format.json { render :show, status: 201 }
+      end
+    else
+      respond_to do |format|
+        errors = {"errors" => @rating_question.errors.messages}
+        format.html { redirect_to "/rating_questions/new", notice: "Must have a title" }
```

Comment:
This action should `render :new`, rather than redirecting back there. The benefit is that it will maintain anything that you've put into the form, but redirecting won't. I can show you what I mean exactly during our next 1-on-1.

Reply:
ok!